package-lock.json

```diff
@@ -6614,9 +6661,9 @@
       "license": "MIT"
     },
     "node_modules/@types/vscode": {
-      "version": "1.97.0",
-      "resolved": "https://registry.npmjs.org/@types/vscode/-/vscode-1.97.0.tgz",
-      "integrity": "sha512-ueE73loeOTe7olaVyqP9mrRI54kVPJifUPjblZo9fYcv1CuVLPOEKEkqW0GkqPC454+nCEoigLWnC2Pp7prZ9w==",
+      "version": "1.99.0",
```

Comment:
Should this not be in sync with our default API version?

Reply:
```
> npm why @types/vscode                                      
@types/vscode@1.99.0 dev
node_modules/@types/vscode
  dev @types/vscode@"^1.50.0" from plugin-a@1.60.0
  sample-plugins/sample-namespace/plugin-a
    plugin-a@1.60.0
    node_modules/plugin-a
      workspace sample-plugins/sample-namespace/plugin-a from the root project
  dev @types/vscode@"^1.50.0" from plugin-b@1.60.0
  sample-plugins/sample-namespace/plugin-b
    plugin-b@1.60.0
    node_modules/plugin-b
      workspace sample-plugins/sample-namespace/plugin-b from the root project
```

This comes from the sample plugins and is only used there. Should probably be aligned, but then we should fix the versions in there, wdyt?

Reply:
@tsmaeder ?

Reply:
Looks fine by me, then.